stackdriver-errors.js

```diff
@@ -14,12 +14,12 @@
  * limitations under the License.
  */
 (function(exports) {
-  "use strict";
```

Comment:
Can you separate these quotes refactoring from the actual change (in 2 different PRs)